src/internal/modules/returns-notifications/controllers/paper-forms.js

```diff
@@ -37,14 +38,18 @@ const roleMapper = require('shared/lib/mappers/role');
  * they wish to send paper return forms
  */
 const getEnterLicenceNumber = async (request, h) => {
-  return h.view('nunjucks/form', {
+  const { licencesWithNoReturns } = request.query;
```

Comment:
Is storing this in request.query the best option? That URL can get quite long...

Could we pop it in the session with Yar or perhaps load it in request.pre?